spec/RedisCacheAdapter.spec.js

```diff
@@ -99,6 +101,155 @@ describe_only(() => {
       .then(value => expect(value).not.toEqual(null))
       .then(done);
   });
+
+  it('redis performance test', async () => {
```

Comment:
might be better to break this down into one `describe` with a before each creating the spys on `this` and then having an `it` for each of the test cases.

Reply:
This test was flaky but I haven't seen it fail in a while. I'll make the changes because this would keep track of bottleneck on the cache